crates/swc_ecma_minifier/tests/fixture/issues/moment/1/output.js

```diff
@@ -910,10 +904,7 @@ function(global, factory) {
         }) : ("string" == typeof input && (config._i = input = config._locale.preparse(input)), isMoment(input)) ? new Moment(checkOverflow(input)) : (isDate(input) ? config._d = input : isArray(format) ? !// date from string and array of format strings
         function(config) {
             var tempConfig, bestMoment, scoreToBeat, i, currentScore, validFormatFound, bestFormatIsValid = !1;
-            if (0 === config._f.length) {
-                getParsingFlags(config).invalidFormat = !0, config._d = new Date(NaN);
-                return;
-            }
+            if (0 === config._f.length) return getParsingFlags(config).invalidFormat = !0, config._d = new Date(NaN);
```

Comment:
This function no longer returns `undefined` but an Invalid Date

Reply:
This is caused by `ignore_return_value`